test/testcafe/spec/Identify_spec.js

```diff
@@ -246,3 +266,27 @@ test.requestHooks(identifyRequestLogger, identifyMockWithFingerprint)('should co
   const factorReqHeaders = factorReq.headers;
   await t.expect(factorReqHeaders['x-device-fingerprint']).notOk();
 });
+
+test.requestHooks(identifyRequestLogger, identifyMockWithFingerprintError)('should continue to compute device fingerprint if there are errors', async t => {
```

Comment:
nit: test is to know if the header is attached right? maybe change the description to 
`adds device fingerprint header and resets submit button if there is an error`?